src/RiveQtQuickItem/riveqsgrhirendernode.cpp

```diff
@@ -399,5 +442,10 @@ void RiveQSGRHIRenderNode::prepare()
     resourceUpdates->updateDynamicBuffer(m_uniformBuffer, 80, 4, &top);
     resourceUpdates->updateDynamicBuffer(m_uniformBuffer, 84, 4, &bottom);
 
-    swapChain->currentFrameCommandBuffer()->resourceUpdate(resourceUpdates);
+    commandBuffer->resourceUpdate(resourceUpdates);
+
+    // postprocess display buffer
+    if (m_postprocessing) {

```

Comment:
at start it crashes here (in windows) because the pointer is not initializes as null